### PR TITLE
Pia 4048 allow configuration of all bank sdk customisation options in the example app

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**
- remove unused configuration options
- reorder properties and methods for better documentation in the `GiniBankConfiguration.swift` file
- cleanup the `GiniBankConfiguration.swift` file

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift [hunk 1]
```diff
@@ -170,282 +198,145 @@ public final class GiniBankConfiguration: NSObject {
...
     /**
-     * Set an adapter implementation to show a custom illustration on the return assistant onboarding page.
+     Set an adapter implementation to show a custom illustration on the "align corners" onboarding page.
      */
-   public var digitalInvoiceOnboardingIllustrationAdapter: OnboardingIllustrationAdapter?
+    public var onboardingAlignCornersIllustrationAdapter: OnboardingIllustrationAdapter?
 
     /**
-     Enable/disable the bottom navigation bar.
+     * Set an adapter implementation to show a custom illustration on the "lighting" onboarding page.
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:211]:
```suggestion
      Set an adapter implementation to show a custom illustration on the "lighting" onboarding page.
```

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift [hunk 2]
```diff
@@ -11,157 +11,185 @@ import GiniBankAPILibrary
...
-
-     - note: Use only if you have a custom `UIAppearance` for your UINavigationBar
-     */
-    @objc public var documentPickerNavigationBarTintColor: UIColor?
-
-    /**
-     Sets the background color of an informal notice. Notices are small pieces of
-     information appearing underneath the navigation bar.
+     Indicates whether the open with feature is enabled or not. In case of `true`,
+     a new option with the open with tutorial wil be shown in the Help menu.
```
[the remark this answers is not in the document]
Comment (zladzeyka) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:54]: And it should be inside `Help screens` pragma below 😉

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift [hunk 3]
```diff
@@ -170,282 +198,145 @@ public final class GiniBankConfiguration: NSObject {
...
+     Should be set if the custom error logging is implemented.
      */
-    public var localizedStringsTableName: String?
+    public var customGiniErrorLoggerDelegate: GiniCaptureErrorLoggerDelegate?
+
+    // MARK: - Development Debug
 
     /**
-     Set dictionary of fonts for available text styles. Used internally.
+     Can be turned on during development to unlock extra information and to save captured images to camera roll.
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift:321]:
```suggestion
     Can be turned on during development to unlock extra information and to save captured images to the camera roll.
```

**Comment by ValentinaIancu-Gini** [comment]
@zladzeyka all changes are implemented.

**Review by zladzeyka — APPROVED** [review]

**Pull request merged**